A1IOSLib/Classes/Ads_Module/Internal Files/AdsAppOpen.swift

```diff
@@ -173,9 +173,10 @@ extension AppOpenAdManager: GADFullScreenContentDelegate {
     _ ad: GADFullScreenPresentingAd,
     didFailToPresentFullScreenContentWithError error: Error
   ) {
-    appOpenAd = nil
```

Comment:
if appOpenAd will be nil then we have to loadads()

Reply:
loadAd() function is already there on line 182.